etc/docker/.env.orchestration.stage

```
@@ -6,15 +6,19 @@ ASPNETCORE_ENVIRONMENT=Development
 DAPR_STATE_STORE_NAME=vnext-state
 DAPR_PUBSUB_STORE_NAME=vnext-pubsub
+DAPR_PUBSUB_BROADCAST_STORE_NAME=vnext-pubsub-broadcast
 DAPR_LOCK_STORE_NAME=vnext-lock
 OTEL_EXPORTER_OTLP_ENDPOINT=http://otel-collector:4317
 OTEL_SERVICE_NAME=vnext-app
 ConnectionStrings__Default=Host=vnext-postgres;Port=5432;Database=Aether_WorkflowDb;Username=postgres;Password=postgres;
 Redis__Standalone__EndPoints__0=vnext-redis:6379
-Telemetry__Otlp__Endpoint=http://otel-collector:4317
-vNextApi__BaseUrl:http://vnext-app:5000
\ No newline at end of file
+Vault__Enabled=false
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The Vault__Enabled key should be in uppercase

(LowercaseKey)